src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,20 @@
 import React from 'react';
+import { User } from '../../types/User';
+import './UserInfo.scss';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+interface Props {
+  user: User,
+}
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const {
+    name,
+    email,
+  } = user;
+
+  return (
+    <a className="UserInfo" href={`mailto:${email}`}>
+      {name}
+    </a>
+  );
+};
```

Comment:
```suggestion
  return (
          <p>
          {' Posted by  '}
           <a className="UserInfo" href={`mailto:${email}`}>
             {name}
           </a> 
        </p>
  );
};
```

Reply:
{' Posted by '} is information about the post, not about the user)